index.html

```diff
@@ -272,7 +277,7 @@ <h3 class="connect-tittle">LET’S CONNECT</h3>
           <h2>Languages</h2>
           <i class="fas fa-chevron-down"></i>
         </div>
-        <ul class="languages-list"  aria-label=" Programming languages" >
+        <ul class="languages-list" aria-label=" Programming languages">
           <li>
             <img src="images/Ellipse-1.png" alt="javaScript logo" />
             <h4>JavaSript</h4>
```

Comment:
Kindly avoid using headings in the middle of the elements such as ul/li to allow accessibility for users using screen readers, instead try using p or span